README.md

```diff
@@ -199,6 +199,9 @@ This can be controlled on a per package basis with the `CPM_DOWNLOAD_<dependency
 ### CPM_USE_LOCAL_PACKAGES
 
 CPM can be configured to use `find_package` to search for locally installed dependencies first by setting the CMake option `CPM_USE_LOCAL_PACKAGES`.
+
+If a packages should always be added the defined way you should set the additional parameter FORCE YES to disable the use of `find_package`
+
```

Comment:
Not sure this is easy to understand. How about instead adding the following at the end of the `CPM_USE_LOCAL_PACKAGES` section (after line 208)?

```markdown
Note that this does not apply to dependencies that have been defined with a truthy `FORCE` parameter. These will be added as defined.
```

Reply:
done